### PR TITLE
Increasing the Loki ingestionRate and ingestionRateBurstSize.

### PR DESCRIPTION
**Pull request by computate**
See the Loki Operator configuration: https://loki-operator.dev/docs/api.md/#ingestionlimitspec
And the loki configuration under the hood: https://grafana.com/docs/loki/latest/configuration/#limits_config

### CODE REVIEW

File: loki/base/lokistacks/lokistack.yaml [hunk 1]
```diff
@@ -4,6 +4,11 @@ metadata:
   name: lokistack
   namespace: openshift-operators-redhat
 spec:
+  limits:
+    global:
+      ingestion:
+        ingestionBurstSize: 10000000
+        ingestionRate: 10000000
```
Comment (joachimweyl) [inline loki/base/lokistacks/lokistack.yaml]: if the units are in MB then is this 10TB, do we need that much space?

Comment (computate) [reply loki/base/lokistacks/lokistack.yaml]: @joachimweyl Thanks for being more careful than I was checking if it was bytes or mb. I have updated it to: 
```yaml
  limits:
    global:
      ingestion:
        ingestionBurstSize: 10
        ingestionRate: 10
```

**Review by joachimweyl — APPROVED** [review]
with new sizing, approved.

**Review by larsks — CHANGES_REQUESTED** [review]
I'd like to see two changes to this pull request:

1. Rebase it down to a single commit.

1. Update the commit message to include an explanation for the change. You could take your message from the slack channel and slightly paraphrase it and I think we would be all set.

**Comment by computate** [comment]
Sure @larsks , I think I'm finally figuring out rebasing my PRs into one commit. The PR is ready for review again, please.

**Review by larsks — APPROVED** [review]
Looks good! In the future, please wrap your commit messages at 70 characters. I've update the message in this PR.

**Pull request merged**